scripts/delete.sh

```diff
@@ -79,3 +79,9 @@ kustomize build default | kubectl delete -f - --ignore-not-found=true
 kustomize build runtimes --load-restrictor LoadRestrictionsNone | kubectl delete -f - --ignore-not-found=true
 kubectl delete -f dependencies/quickstart.yaml --ignore-not-found=true
 kubectl delete -f dependencies/fvt.yaml --ignore-not-found=true
+
+# Roll back to previous status
+if [[ -n $namespace ]]; then
+  kubectl config set-context --current --namespace="default"
+fi
+kubectl delete ns $namespace
```

Comment:
We don't create namespace in install.sh; therefore probably should not delete namespace automatically in delete.sh. The namespace is created outside of our scripts as instructed here, https://github.com/kserve/modelmesh-serving/blob/main/docs/install/install-script.md. Maybe consider to add a line of instruction in the "Delete the installation" section to manually delete namespace just in case the user wants to keep anything in the namespace after uninstall modelmesh-serving?

Reply:
I believe the delete ns command will error out when the optional namespace is not provided.